### PR TITLE
Register Subdomain: worm

### PR DESCRIPTION
**Pull request by korn176**
Added the domain: worm

### CODE REVIEW

File: domains/worm.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "arq091",
+        "email": "poonyapat.thane@gmail.com"
+      },
+      "record": {
+        "CNAME": "https://arq091.github.io/worm"
```
Comment (WilliamDavidHarrison) [inline domains/worm.json:8]:
```suggestion
        "CNAME": "arq091.github.io"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]
Also I can't reach the website.

File: domains/worm.json (hunk 1 above)
Comment (kornls) [reply domains/worm.json:8]: Ah sorry, incorrect link.

File: domains/worm.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "arq091",
+        "email": "poonyapat.thane@gmail.com"
+      },
+      "record": {
+        "CNAME": "arq091.github.io/wiggly-worm"
```
Comment (WilliamDavidHarrison) [inline domains/worm.json:8]:
```suggestion
        "CNAME": "arq091.github.io"
```

The wriggly worm website will still show up even when the `CNAME` doesn't have the URL.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by kornls** [comment]
Changes have been done accordingly.

**Review by wdhdev — APPROVED** [review]

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**